ng/scriptGenerators/cli/resources/GernericResource.ts

```diff
@@ -0,0 +1,28 @@
+﻿module armExplorer {
+
+    export class GenericResource implements ICliResource {
+
+        constructor(protected resolver: ScriptParametersResolver) {
+        }
+
+        public getScript(action: ResourceAction): String {
+            switch (action) {
+            case ResourceAction.Get:
+                return `az resource show --id ${this.resolver.getCompleteResourceId()} --api-version ${this.resolver.getParameters().apiVersion}\n\n`;
```

Comment:
It looks like all the "scripts" have to end with `\n\n`. Can this be centralized somewhere?